e3sm_to_cmip/cmor_handlers/handlers.yaml

```diff
@@ -670,14 +670,6 @@
   formula: null
   positive: up
   levels: null
-- name: rsutcs
-  units: W m-2
-  raw_variables: [SOLIN, FSNTOAC]
-  table: CMIP6_Amon.json
-  unit_conversion: null
-  formula: SOLIN - FSNTOAC
-  positive: up
-  levels: null
```

Comment:
`rsutcs` was added through this recent PR by @chengzhuzhang in Sep/2023.

https://github.com/E3SM-Project/e3sm_to_cmip/pull/210

> FSUTOA and FSUTOAC are used in e3sm_to_cmip for directly getting rsut and rsutcs. For v3 output, both FSUTOA and FSUTOAC are removed. This PR is to update formula to use available variables, i.e., rsut = SOLIN - FSNTOA, and rsutcs = SOLIN - FSNTOAC

Not sure if it should be removed?

Reply:
@TonyB9000 comment on the reason for this change: https://github.com/E3SM-Project/e3sm_to_cmip/issues/251#issuecomment-2013025956

Reply:
No, we should not remove this entry.

Reply:
@chengzhuzhang Same issue:  If there are multiple ways (v2 versus v3) to calculate `rsutcs` how is the correct handler selected?

I cannot believe the user (batch process) must seek a selection of variable-sets in the native data, just to discover if there is a match.  That is just wrong.  If the handlers are "model-version specific", they need to have the model-version as a searchable attribute.